docs/pages/docs/setup.md

```diff
@@ -97,10 +97,11 @@ After running either command, the `database.g.dart` file containing the generate
 class will have been generated.
 You will now see errors related to missing overrides and a missing constructor. The constructor
 is responsible for telling drift how to open the database. The `schemaVersion` getter is relevant
-for migrations after changing the database, we can leave it at `1` for now. The database class
-now looks like this:
-<a name="open">
-{% include "blocks/snippet" snippets = snippets name = 'open' %}
+for migrations after changing the database, we can leave it at `1` for now. Update `database.dart`
+so it now looks like this:
+
+<a name="after_generation">
```

Comment:
This link seems to cause Firefox to parse the HTML in a weird way, it breaks the snippet below for me. I guess we need to add a closing `</a>` immediately afterwards, I don't know how this has ever worked...

Still, the anchor should be named `open` because there are other parts that might link to that. So let's keep make it

```suggestion
<a name="open"></a>
```

Reply:
suggested change applied